iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/DateTimeUtils.java

```diff
@@ -665,17 +665,21 @@ public static long currentTime() {
     }
   }
 
-  public static String convertLongToDate(long timestamp) {
-    String timePrecision = CommonDescriptor.getInstance().getConfig().getTimestampPrecision();
-    switch (timePrecision) {
+  public static String convertLongToDate(long timestampInMs) {
+    return convertLongToDate(
+        timestampInMs, CommonDescriptor.getInstance().getConfig().getTimestampPrecision());
+  }
+
+  public static String convertLongToDate(long timestampInMs, String targetPrecision) {
```

Comment:
It seems that the name of timestampInMs is not correct?

Reply:
Fixed.